curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/6401d73eba6cdb4c42f9b134.md

```diff
@@ -0,0 +1,166 @@
+---
+id: 6401d73eba6cdb4c42f9b134
+title: Step 21
+challengeType: 0
+dashedName: step-21
+---
+
+# --description--
+
+Now, call the `displayAuthors` function just like you did the other time.
+
+If you click the "Load More Authors" after calling the function, it still won't work. That's because you are yet to add the click event listener to the button. We'll do that next.
```

Comment:
We will need to remove all references to first person narrative per the [docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions) here